adapter/src/main/java/com/twilio/kudu/sql/rel/KuduNestedJoin.java

```diff
@@ -92,25 +92,10 @@ public KuduNestedJoin copy(final RelTraitSet traitSet, final RexNode condition,
 
   @Override
   public RelOptCost computeSelfCost(final RelOptPlanner planner, final RelMetadataQuery mq) {
-    final double rowCount = mq.getRowCount(this);
```

Comment:
I thought we were going to build on this.
Is this a goner ?

Reply:
It wasn't ever used. Our row calculations are bad atm. We are not using the modern Kudu Client to get the row counts. 

I would also like to update `KuduToEnumerableRel` to do use the right `RexXXX` variables for the existing `EnumerableNestedBatchJoin` so we can remove this rule *and the `KuduNestedJoin` rel*.

Reply:
I think these changes were done poorly and I think you were right to call out these changes.

This change produces some changes to the plans. notably for some of the `HashJoins(type=inner)` on `ActorDimension` changed to `KuduNestedLoop`. 

Going to play around and back this change out a bit.